examples/rllib/tests/test_environment.py

```diff
@@ -0,0 +1,55 @@
+from meltingpot.python import substrate
+from examples.rllib.utils import env_creator
+from gym.spaces.discrete import Discrete
+from meltingpot.python.configs.substrates.commons_harvest_open import ACTION_SET
+
+# Setup an environment to test
+SUBSTRATE_NAME = "commons_harvest_open"
+env_config = substrate.get_config(SUBSTRATE_NAME)
+
+
```

Comment:
Could you please use the Google testing library to write these tests? That would make them consistent with the rest of the tests: e.g. https://github.com/deepmind/meltingpot/blob/main/meltingpot/python/substrate_test.py

Reply:
Yes no problem at all - I'll switch those across